scripts/docgen/snippets/writeFilesConsumer.ts

```diff
@@ -0,0 +1,27 @@
+import { writeFileSync } from "node:fs"
+import { join } from "node:path"
+import { DocGenSnippetConsumer } from "../config.js"
+
+export type CreateWriteFilesConsumerOptions = {
+    rootOutDir: string
+    transformFileName?: (name: string) => string
```

Comment:
Yeah, this should definitely have been `transformRelativePath?: (path: string) => string`